scenarios/goldilocks.benchmarks.yml

```diff
@@ -37,9 +48,38 @@ scenarios:
 
   stage1vanilla:
     application:
-      job: aspnetbenchmarks
+      job: stage1aspnetbenchmarks
     load:
       job: wrk
       variables:
         presetHeaders: json
-        path: /todos
\ No newline at end of file
+        path: /todos
+
+  stage1grpcpublishaot:
+    application:
+      job: stage1grpcaspnetbenchmarks
+      buildArguments: 
+        - "/p:PublishAot=true"
+        - "/p:StripSymbols=true"
+    load:
+      job: h2LoadClient
+      variables:
+        presetHeaders: grpc
+        path: /goldilocks.TodoService/GetAllTodos
+        body: AAAAAAA=
+        connections: 28
+        threads: 28
+        streams: 70
```

Comment:
Someday we should do another pass at running a set of benchmarks and vary clients/streams to find the sweet spot.